cardano-api/test/Test/Cardano/Api/Typed/Value.hs

```diff
@@ -65,7 +69,21 @@ canonicalise =
     isZeroOrEmpty (ValueNestedBundleAda q) = q == 0
     isZeroOrEmpty (ValueNestedBundle _ as) = Map.null as
 
+
+prop_roundtrip_AssetName_JSON :: Property
+prop_roundtrip_AssetName_JSON =
+  property $ do
+    v <- forAll genAssetName
+    tripping v encode eitherDecode
+
+prop_roundtrip_AssetName_JSONKey :: Property
+prop_roundtrip_AssetName_JSONKey =
+  property $ do
+    v <- forAll genAssetName
+    tripping (Map.singleton v ()) encode eitherDecode
+
+
 -- -----------------------------------------------------------------------------
 
 tests :: TestTree
-tests = fromGroup $$discover
+tests = $testGroupGenerator
```

Comment:
Why the change to `testGroupGenerator`?

Reply:
There is already grouping ability in `tasty` library, we don't need our home-grown version.

Reply:
Indeed, this change is lateral, reverted. It should be done more generally in a separate PR.